avm/res/sql/server/database/main.bicep

```
@@ -223,8 +225,16 @@ resource database 'Microsoft.Sql/servers/databases@2023-08-01' = {
     elasticPoolId: elasticPoolResourceId
     encryptionProtector: customerManagedKey != null
```

```suggestion
    encryptionProtector: !empty(customerManagedKey}
```
I think we use this all over the place - also because `{}` would otherwise not be interpreted as false